### PR TITLE
Chat: add Overview demo description

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: apps/demos/Demos/Chat/Overview/description.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Chat is an interactive interface that allows users to send and receive messages in real time.
+
+To get started with the DevExtreme Chat component, refer to the following step-by-step tutorial: [Getting Started with Chat](/Documentation/Guide/UI_Components/Chat/Getting_Started_with_Chat/).
+
+The demo covers Chat basics: setting initial messages, displaying new messages, managing users, and linking two chats in real-time.
```
Comment (Abadzhev) [inline apps/demos/Demos/Chat/Overview/description.md]:
```suggestion
The demo implements basic Chat functionality: specifies initial messages, updates the conversation with new incoming and outgoing messages, manages users, and links two chats in real-time.
```

File: apps/demos/Demos/Chat/Overview/description.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+<!--split-->
+
+## Messages
+
+To set initial messages, you can either populate the [items](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#items) array (shown in this demo) or use [dataSource](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#dataSource).
```
Comment (Abadzhev) [inline apps/demos/Demos/Chat/Overview/description.md]:
```suggestion
To specify initial messages, you can either populate the [items](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#items) array (shown in this demo) or use a [dataSource](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#dataSource).
```

File: apps/demos/Demos/Chat/Overview/description.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+To render a new message, you must:
```
Comment (Abadzhev) [inline apps/demos/Demos/Chat/Overview/description.md]:
```suggestion
Use the following API to render new messages:
```

File: apps/demos/Demos/Chat/Overview/description.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+- If you use **items**, update the array with the new message.
+- If using **dataSource**, implement [load](/Documentation/ApiReference/Data_Layer/CustomStore/Configuration/#load) and [insert](/Documentation/ApiReference/Data_Layer/CustomStore/Configuration/#insert) operations.
```
Comment (Abadzhev) [inline apps/demos/Demos/Chat/Overview/description.md]: I want to keep list items consistent. Don't want one to read "If you use" and another "If using". 

```suggestion
- If you use a **dataSource**, implement [load](/Documentation/ApiReference/Data_Layer/CustomStore/Configuration/#load) and [insert](/Documentation/ApiReference/Data_Layer/CustomStore/Configuration/#insert) operations.
```

File: apps/demos/Demos/Chat/Overview/description.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+## Users
+
+To specify the owner, assign a user to the Chat [user](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#user) property. Owner messages align to the right (or left in RTL mode), without displaying the owner's name and avatar.
```
Comment (Abadzhev) [inline apps/demos/Demos/Chat/Overview/description.md]:
```suggestion
To specify the chat owner, set the [user](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#user) property. Owner messages align to the right (or left in RTL mode) and do not display the name or avatar.
```

File: apps/demos/Demos/Chat/Overview/description.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+Each message includes information about the sender ([author](/Documentation/ApiReference/UI_Components/dxChat/Types/Message/#author)), and includes name, avatar, and alt avatar text. If no avatar is set, the user's initials are displayed instead.
```
Comment (Abadzhev) [inline apps/demos/Demos/Chat/Overview/description.md]:
```suggestion
Each message includes information about the sender ([author](/Documentation/ApiReference/UI_Components/dxChat/Types/Message/#author)): name, avatar, and alternative avatar text. If no avatar is set, the user's initials are displayed instead.
```

File: apps/demos/Demos/Chat/Overview/description.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+## Events
+
+Entering a message triggers the [messageEntered](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#onMessageEntered) event. Use the event handler to perform message post processing (like displaying the message in a message feed and sending the message to the server for storage).
```
Comment (Abadzhev) [inline apps/demos/Demos/Chat/Overview/description.md]:
```suggestion
If a user enters a message, the Chat raises the [messageEntered](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#onMessageEntered) event. Use the event handler to process the message. For example, you can simply display the message in the message feed and send the message to the server for storage.
```

File: apps/demos/Demos/Chat/Overview/description.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+
+Start typing to raise [typingStart](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#onTypingStart), and stop typing or enter a message into the Chat for [typingEnd](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#onTypingEnd). By using these event handlers, you can manage the [typingUsers](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#typingUsers) array (it shows which users are typing in the chat UI).
```
Comment (Abadzhev) [inline apps/demos/Demos/Chat/Overview/description.md]:
```suggestion
When users start or finish typing, the Chat raises [typingStart](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#onTypingStart) and [typingEnd](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#onTypingEnd) events. Use these events to manage the [typingUsers](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#typingUsers) array. The Chat uses this array to display a list of active users.
```

File: apps/demos/Demos/Chat/Overview/description.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+
+## Users
+
+To specify the chat owner, set the [user](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#user) property. Owner messages align to the right (or left in RTL mode) and do not display the name or avatar.
+
+Each message includes information about the sender ([author](/Documentation/ApiReference/UI_Components/dxChat/Types/Message/#author)): name, avatar, and alternative avatar text. If no avatar is set, the user's initials are displayed instead.
+
+## Events
+
+If a user enters a message, the Chat raises the [messageEntered](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#onMessageEntered) event. Use the event handler to process the message. For example, you can simply display the message in the message feed and send the message to the server for storage.
```
Comment (dirk-pieterse) [inline apps/demos/Demos/Chat/Overview/description.md]:
```suggestion
If a user enters a message, the Chat raises the [messageEntered](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#onMessageEntered) event. Use the event handler to process the message. For example, you can display the message in the message feed and send the message to the server for storage.
```

File: apps/demos/Demos/Chat/Overview/jQuery/description.md [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+
+## Users
+
+To specify the chat owner, set the [user](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#user) property. Owner messages align to the right (or left in RTL mode) and do not display the name or avatar.
+
+Each message includes information about the sender ([author](/Documentation/ApiReference/UI_Components/dxChat/Types/Message/#author)): name, avatar, and alternative avatar text. If no avatar is set, the user's initials are displayed instead.
+
+## Events
+
+If a user enters a message, the Chat raises the [messageEntered](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#onMessageEntered) event. Use the event handler to process the message. For example, you can simply display the message in the message feed and send the message to the server for storage.
```
Comment (dirk-pieterse) [inline apps/demos/Demos/Chat/Overview/jQuery/description.md]:
```suggestion
If a user enters a message, the Chat raises the [messageEntered](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#onMessageEntered) event. Use the event handler to process the message. For example, you can display the message in the message feed and send the message to the server for storage.
```

**Review by dirk-pieterse — DISMISSED** [review]

File: apps/demos/Demos/Chat/Overview/jQuery/description.md [hunk 11]
```diff
@@ -0,0 +1,27 @@
+CChat is an interactive interface that allows users to send and receive messages in real time.
```
Comment (vladaskorohodova) [inline apps/demos/Demos/Chat/Overview/jQuery/description.md]:
```suggestion
Chat is an interactive interface that allows users to send and receive messages in real time.
```

**Review by EugeniyKiyashko — APPROVED** [review]

**Pull request merged**